channel.go

```diff
@@ -199,15 +220,15 @@ func isAFVSockSupported() (bool, error) {
 }
 
 func vSockPathExist() (bool, error) {
```

Comment:
The way you modified this function, there's no need for `vSockPathExist()` to return an error anymore. It should be a simple boolean.

Reply:
you're right